wasm_test.go

```diff
@@ -396,7 +401,7 @@ var _ = Describe("WASM Modules", func() {
 			Expect(wasmResp.OutputPayload).Should(MatchJSON(`{"object": {"type": "str", "cc_num": "1234"}}`))
 		})
 
-		It("truncates a field by total length", func() {
+		It("truncates a field by percent length", func() {
```

Comment:
Fixing duplicate test, we already have total length but needed a percent length also